### DISCUSSION TITLE
Could be there a patch with the vulnerability fix for `yaml@1.10.2` ?

### DISCUSSION THREAD

**Discussion by eppisapiafsl** [eemeli/yaml#469] [Q&A] [answered]
Hi! 

Wondering if you plan to do a patch to fix the vulnerability in Yaml@1.10.2 so we can apply a resolution to transitive dependencies installing the old version 🙏

**Comment by eemeli** [comment, marked as answer]
`yaml@1` is not affected by this vulnerability, and does not need to be patched. A fresh `npm audit` should show it passing. The version range has also been updated at least on GitHub: https://github.com/advisories/GHSA-f9xv-q969-pqx4

**Reply by eppisapiafsl** [reply]
Thanks! Yeah, I just notice dependabot changed the issue 🚀
